packages/vite/types/shims.d.ts

```diff
@@ -79,7 +79,7 @@ declare module 'postcss-modules' {
 declare module '@rollup/plugin-dynamic-import-vars' {
   import { Plugin } from 'rollup'
 
-  interface Options {
+  export interface Options {
```

Comment:
It's not imported anywhere else, why export it?

Reply:
I don't remember. I think this is just a leftover from when I played around with it... I removed it